calma_grid/models/product_template.py

```diff
@@ -70,35 +74,39 @@ def _check_wallet(self):
             raise ValidationError(
                 _('Primero debe crear una compañía con MarketpayId'))
 
-    @api.model
-    def create(self, values):
-        product = super().create(values)
-        marketpay_domain = product.company_id.marketpay_domain
-        token_url = product.company_id.token_url
-        key = product.company_id._prepare_marketpay_key()
-        data = {'grant_type': 'client_credentials'}
-        headers = {'Authorization': key,
-                   'Content-Type': 'application/x-www-form-urlencoded'}
-        r = requests.post(token_url, data=data, headers=headers)
-        rs = r.content.decode()
-        response = json.loads(rs)
-        token = response['access_token']
-        config = swagger_client.Configuration()
-        config.host = marketpay_domain
-        config.access_token = token
-        swagger_client.ApiClient(configuration=config)
-        swagger_client.Configuration.set_default(config)
-        swagger_client.UsersApi()
+    @api.constrains('crowdfunding')
+    def _create_wallet(self):
+        self._check_wallet()
+        if self.crowdfunding == True:
+            if self.project_wallet == False:
```

Comment:
En python eso se escribe así:

```suggestion
            if self.cowdfunding and not self.project_wallet:
```